app/electron/port-forward.ts

```diff
@@ -0,0 +1,52 @@
+import * as k8s from '@kubernetes/client-node';
+import { dialog, ipcMain } from 'electron';
+import { BrowserWindow } from 'electron/main';
+import * as net from 'net';
+
+export function handlePortForward(
+  mainWindow: BrowserWindow,
+  kubeconfig,
+  namespace: string,
+  podname: string,
+  targetPort: number
+) {
+  const kc = new k8s.KubeConfig();
+  if (kubeconfig) {
+    kc.loadFromFile(kubeconfig);
+  } else {
+    kc.loadFromDefault();
+  }
+  const forward = new k8s.PortForward(kc);
+  mainWindow.webContents.send('port-forward-initiated', {
+    podname,
+    namespace,
+    cluster: kc.getCurrentCluster()?.name,
+  });
+  const server = net.createServer(socket => {
+    console.log('port forward initiated request');
```

Comment:
Please log with more info so we know which ports/pods are requesting it.

Reply:
this is no longer needed as we now use the backend api